.eslintrc

```
@@ -0,0 +1,18 @@
+{
+    "root": true,
+    "extends": ["plugin:@18f/eslint-plugin-identity/recommended"],
+    "plugins": ["@18f/eslint-plugin-identity"],
+    "env": {
+      "browser": true,
+      "jest": true
```

I don't think we'll need `jest` here since we're not using it (and frankly I hope we won't!)

```suggestion
      "browser": true
```